package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/user-event": "^12.8.3",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-router-dom": "^6.0.2",
```

Comment:
I really love this new version of Router.  Wasn't sure if I would, but it has a much cleaner feel.  I like clean things.